core/page-assets/loader.php

```diff
@@ -24,7 +24,13 @@ public function get_name() {
 
 	private function init_assets() {
 		$this->assets = [
-			'styles' => [],
+			'styles' => [
+				'e-animations' => [
```

Comment:
Remove `wp_register_style` of animations

Reply:
Done.